openg2p_disbursement/models/openg2p_disbursement_slip.py

```diff
@@ -618,3 +621,68 @@ def get_disbursement_line_total(self, code):
             return line[0].total
         else:
             return 0.0
+
+    def create_bulk_transfer(self):
+        query = """SELECT p.total, p.currency_id, c.sanitized_acc_number
+                        FROM public.openg2p_disbursement_advice AS p  
+                        LEFT JOIN public.openg2p_disbursement_advice FROM public.res_partner_bank AS c  
+                        ON p.id=c.id  
+                        INTO OUTFILE 'accounts.csv' """
```

Comment:
What are the CSV file headers?

Reply:
Thsi is the earlier written code i need to comment it out.All the logic for single and batch transaction is in openg2p_disbursement_batch_transaction,openg2p_disbursement_single_transaction,openg2p_disbursement_main.